src/logmask.cpp

```diff
@@ -12,6 +12,14 @@
 #include "unc_ctype.h"
 
 
+/*
+ * the test suite Coveralls: https://coveralls.io
+ * will complains because these functions are only
+ * used at developement time.
+ * Don't worry about unsed lines for the functions:
+ *   logmask_to_str
+ *   logmask_from_string
```

Comment:
Really? It seems `logmask_from_string` should be called any time `-L` is used, and we *definitely* have tests for that... (`logmask_to_str` OTOH is totally unused. How likely is it to actually be useful?)